### ISSUE TITLE
Any clues as to how I might go about bundling this for Android?

### ISSUE THREAD

**Issue by AdityaAnand1** [gl-transitions/gl-transitions#129]
Would it be possible to somehow bundle this into an ARM native library?

**Comment by divysoni3** [comment]
@AdityaAnand1  have u find any clue to use them in android?

**Comment by gre** [comment]
you need to write your own implementation, implement a GL Surface with some shaders and hook the transition parameters. there is not a library for it but feel free to contribute to the space :)

- this JS code can help as reference https://github.com/gre/gl-transition-libs/blob/master/packages/gl-transition/src/index.js but there are magic done under the hood by library "gl-shader". also WebGL is a tiny bit more high level than OpenGL, so it needs to be translated to OpenGL code.
- this code as reference can also help, it's a C++ implementation that use gl-transition: https://github.com/transitive-bullshit/ffmpeg-gl-transition

to summarize, it's not such a big deal to implement, but i'm not exactly sure what kind of library this needs to be (does it just expose utilities, does it expose a Surface?) and it might all depend on the actual usecase
